opentracing-api/src/main/java/io/opentracing/ActiveSpan.java

```diff
@@ -33,6 +33,42 @@
  * {@link ActiveSpan.Continuation}s, then re-{@link Continuation#activate()}d later.
```

Comment:
I think this PR tries to distance from refcounting. There's a line that still talks about that which might be something to remove in this PR https://github.com/bhs/opentracing-java/blob/91cf7e5e500f635b9b765df3a107e0de32ad5f0e/opentracing-api/src/main/java/io/opentracing/ActiveSpan.java#L108